tests/test_web_protocol.py

```diff
@@ -362,7 +362,7 @@ def close():
         b'GET / HTTP/1.1\r\n'
         b'Host: example.com\r\n'
         b'Content-Length: 0\r\n\r\n')
-    await asyncio.sleep(0)
+    await asyncio.sleep(0.01)
```

Comment:
Doesn't seem related.

Reply:
Otherwise some tests are failing, my guess it's because I had to await one more task, so 0 is not enough.